tasks/extract_filepaths_for_granule/test/unit_tests/test_extract_file_paths_for_granule.py

```diff
@@ -166,21 +166,21 @@ def test_task_no_filepath(self):
         self.task_input_event["config"]["private-bucket"] = "my_private_bucket"
```

Comment:
Should have an additional test here to run against your schemas+CumulusMessageAdapter. See test_handler_happy_path in request_files for an example.

Reply:
I agree, you should also have a test for when there is no excludeFileTypes values.

Reply:
Once you have a happy path working, you can try tweaking a key in the input or output and have another test with an expected failure when the data doesn't match the schema.

Reply:
yes added test to check that in `test_task_input_schema_return_error()`

Reply:
You need to run it through CMS via RunCumulusTask to get a realistic test. There's a lot of internal translation and modification in that class. Also, still need a happy_path.

Reply:
added that test to run through CMS
`https://github.com/nasa/cumulus-orca/pull/126/commits/e3ffca84e241ae1c60e06c3408726980a8aa39f0`